app/init_cmd.go

```diff
@@ -7,8 +7,8 @@ import (
 )
 
 type initCmd struct {
-	NoGit   bool     `help:"Disable Hermit's automatic management of Git'"`
-	Idea    bool     `help:"Enable Hermit's automatic addition of its IntelliJ IDEA plugin"`
+	Git     bool     `negatable:"" help:"Disable Hermit's automatic management of Git'"`
```

Comment:
Should this be `Enable` now? Similar to `Idea`
```suggestion
	Git     bool     `negatable:"" help:"Enable Hermit's automatic management of Git'"`
```